index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portofolio-Desktop-Version</title>
+  </head>
+  <body>
+      <header class="nav-bac backgroun-color">
+        <h1><a href="#" class="my-logo">MRKA</a></h1>
+        <nav class="nav-bar">
+          <ul class="menu">
+            <li><a href="#parent-cards">Portofolio</a></li>
+            <li><a href="#about-content">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <article id="page-header">
+          <section class="page-header-container">
+            <h2>I'm Mohammad Rafi Amin <br>Glad to see you!</h2>
+            <p>
+              I’m a software developer! I can help you build a product , feature or website Look through<br>some of my work and experience! If you like what you see and have a project you need<br>coded, don’t hestiate to contact me.
+            </p>
+            <h4 class="text-social">LETS's CONNECT</h4>
+            <ul class="contacts-social">
```

Comment:
- [ ] You did very well implementing all that te previous requested. Nice design! 👏 
However, I noticed that the mobile version is broken after you implement the desktop version

See your design on `375px` view. Kindly address it
![image](https://user-images.githubusercontent.com/84156714/193368808-bcbbcfae-5f74-4bfc-b46f-0484101a19ec.png)

![image](https://user-images.githubusercontent.com/84156714/193368744-5dc2ae53-f68d-431d-b3cb-5f2482157d61.png)

Reply:
@bobb-Rob 
thanks for your good and valuable feedback. 
you mention the very important thing it will be fixed in my next trying.